services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/BeanTableSchema.java

```diff
@@ -222,12 +224,23 @@ public TableMetadata tableMetadata() {
         return builder.build();
     }
 
-    private static Optional<AttributeConverterProvider> converterProviderAnnotation(DynamoDbBean dynamoDbBean) {
-        Class<?>[] converterClasses = dynamoDbBean.converterProviders();
-        //TODO: temporary solution to pick one AttributeConverterProvider.
-        return converterClasses.length > 0 ?
-               Optional.of((AttributeConverterProvider) newObjectSupplierForClass(converterClasses[0]).get()) :
-               Optional.empty();
+    private static List<AttributeConverterProvider> converterProviderAnnotation(DynamoDbBean dynamoDbBean) {
```

Comment:
I chose the name because of the prior art of naming similar methods in the same class, called from that particular method. I am somewhat on the same page as you, but then I'd rename all of the methods.

Reply:
Yes feel free to rename anything that doesn't make sense, I was just looking at your changes.